### PR TITLE
Maintaining the consistency of tests among master and release branch

### PR DESCRIPTION
**Pull request by Anipik**
Related Release PR https://github.com/dotnet/corefx/pull/31046

### CODE REVIEW

**Comment by Anipik** [comment]
@dotnet-bot test Packaging All Configurations x64 Debug Build

**Comment by Anipik** [comment]
@danmosemsft should this also wait for yout change regarding driveFormat ?

**Comment by danmosemsft** [comment]
@Anipik can you fix it (and the original millisec test) please? Just don't get DriveFormat if PlatformDetection.IsInAppContainer. I suggest you fix the tests in master, then port all that to 2.1.

**Comment by Anipik** [comment]
@danmosemsft yeah sure i am on it

File: src/System.IO.FileSystem/tests/Base/BaseGetSetTimes.cs [hunk 1]
```diff
@@ -71,50 +76,8 @@ public void CanGetAllTimesAfterCreation()
             ValidateSetTimes(item, beforeTime, afterTime);
         }
 
-        [Fact]
-        [PlatformSpecific(TestPlatforms.AnyUnix)] // Windows tested below, and OSX HFS driver format does not support millisec granularity
+        [ConditionalFact(nameof(isNotHFS))] // Windows tested below, and OSX HFS driver format does not support millisec granularity
```
Comment (danmosemsft) [inline src/System.IO.FileSystem/tests/Base/BaseGetSetTimes.cs]: Windows is no longer "tested below"

**Pull request merged**